values/binary.go

```diff
@@ -22,15 +22,15 @@ func LookupBinaryFunction(sig BinaryFuncSignature) (BinaryFunction, error) {
 	if !ok {
 		return nil, fmt.Errorf("unsupported binary expression %v %v %v", sig.Left, sig.Operator, sig.Right)
 	}
-	return binaryFuncNullCheck(f), nil
+	return binaryFuncNullCheck(f, sig), nil
 }
 
 // binaryFuncNullCheck will wrap any BinaryFunction and
 // check that both of the arguments are non-nil.
 //
 // If either value is null, then it will return null.
 // Otherwise, it will invoke the function to retrieve the result.
-func binaryFuncNullCheck(fn BinaryFunction) BinaryFunction {
+func binaryFuncNullCheck(fn BinaryFunction, sig BinaryFuncSignature) BinaryFunction {
```

Comment:
Why is this added to the function signature?

Reply:
I added it originally. This is a mistake that was kept when I modified things.